Marketplace/repositories/usuarioRepository.js

```diff
@@ -9,19 +9,10 @@ module.exports = {
 			include: [ "roles" ]
 		});
 	},
-	ObtenerPorEmail: async (email) => {
-		const usuarios = await entidad.findAll({
+	ObtenerPorEmail: (email) => {
+		return entidad.findOne({
 			where: {email: email}
 		});
-
-		return usuarios[0];
-	},
-	EmailYaExistente: async (email, id) => {
-		let cantidad = await entidad.count({
-			where: {email: email}
-		});
-
-		return cantidad > 0;
 	},
```

Comment:
Se eliminó este método, porque:
1. No cumplía su fiunción, ya que tenía como parámetro un ID que no se usaba.
2. Tendría sentido especialmente para EDITAR USUARIO, pero en mi caso no sé usar la opción "not equal" para el ID.
3. Para LOGIN y REGISTRO no es imprescindible, ya que se puede usar el método _ObtenerPorEmail_ con el mismo fin.